src/index.mjs

```diff
@@ -33,6 +34,9 @@ export default class SizePlugin {
 		this.options = options || {};
 		this.pattern = this.options.pattern || '**/*.{mjs,js,css,html}';
 		this.exclude = this.options.exclude;
+		this.history = this.options.history;
+		this.historyPath = path.resolve(this.options.historyPath || process.cwd(), 'build-sizes.json');
```

Comment:
```suggestion
		this.historyPath = path.resolve(process.cwd(), this.options.filename || 'build-sizes.json');
```

Also I think if this moves into `.apply()` we could use `compiler.context` instead of `process.cwd()`.